app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,273 @@
+    <string name="title_activity_result_send_email">E-mail aangetroffen</string>
+    <string name="title_activity_result_product">Product aangetroffen</string>
+    <string name="title_activity_result_tel">Telefoonnummer aangetroffen</string>
+    <string name="title_activity_result_wifi">Wi-Fi aangetroffen</string>
+    <string name="title_activity_result_contact">Contactpersoon aangetroffen</string>
+    <string name="action_help">Hulp</string>
+
+    <!-- ### WELCOME DIALOG ### -->
+    <string name="welcome_text">Met de PRIVACYVRIENDELIJKE QR-codescanner-app kun je barcodes en QR-codes met vari?rende inhoud scannen. Deze app vereist nauwelijks machtigingen en heeft ondersteuning voor het detecteren van kwaadaardige links.</string>
+    <string name="welcome">Welkom</string>
+    <string name="viewhelp">Hulp tonen</string>
+
+    <!--<string name="help_privacy_heading">Privacy Info</string>
+    <string name="about_author_contributors">Contributors</string>
+    <string name="privacy_friendly">This application belongs to the group of PRIVACY FRIENDLY Apps developed by Technische Universität Darmstadt. Sourcecode licensed under GPLv3. Images copyright TU Darmstadt and Google Inc.</string>
+    <string name="more_info">More information can be found on:</string>
+    <string name="github" translatable="false"><a href="https://github.com/SecUSo/privacy-friendly-qr-scanner">Github-Repo</a></string>-->
+    <string name="help_permission1_description">Deze app vereist de cameramachtiging om barcodes and QR-codes te kunnen scannen middels de camera.</string>
+    <string name="help_permission1_heading">Camera</string>
+    <string name="help_privacy_heading">Welke machtigingen vereist de PRIVACYVRIENDELIJKE QR-scanner?</string>
+    <string name="help_category_example">Het scannen werkt niet. Wat kan ik hieraan doen?</string>
+    <string name="help_overview_heading">Wat is de PRIVACYVRIENDELIJKE QR-scanner?</string>
+    <string name="help_intro">Met de PRIVACYVRIENDELIJKE QR-codescanner-app kun je barcodes en QR-codes met vari?rende inhoud scannen.</string>
+    <string name="help_whatis_answer">Met de PRIVACYVRIENDELIJKE QR-codescanner-app kun je barcodes en QR-codes met vari?rende inhoud scannen en genereren. Daarnaast kun je gescande QR-codes kopi?ren naar of delen met andere apps. Ook kun je gegeneerde QR-codes opslaan in de galerij en delen.</string>
+
+
+    <!--###HELP###-->
+
+
+    <string name="help_whatis">Wat is de PRIVACYVRIENDELIJKE QR-scanner?</string>
+
+    <string name="help_usability">Het scannen werkt niet. Wat kan ik hieraan doen?</string>
+    <string name="help_usability_answer">Als je geen codes kunt scannen, vergroot dan de afstand tussen je apparaat en de code.</string>
+    <string name="help_privacy">Wat maakt de app privacyvriendelijk?</string>
+    <string name="help_privacy_answer">Privacy Friendly App-voorbeeld vereist geen enkele machtiging. Daarnaast wordt er niks bijgehouden en geen reclame getoond.</string>
```

Comment:
correctly translated, but source was wrong, 
```suggestion
    <string name="help_privacy_answer">Privacy Friendly QR-scanner vereist geen enkele machtiging. Daarnaast wordt er niks bijgehouden en geen reclame getoond.</string>
```
Translation is fine, but source has wrong App name.